examples/switch_mode_persist.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/python3
+
+# Switch between modes without reallocating buffers.
+
+import time
+
+from picamera2 import Picamera2, Preview
+from picamera2.allocators import PersistentAllocator
+
+picam2 = Picamera2()
+picam2.allocator = PersistentAllocator()
+picam2.start_preview(Preview.QTGL)
+
+# Create preview configuration
+preview_config = picam2.create_preview_configuration()
+picam2.configure(preview_config)
+
+size = picam2.sensor_resolution
+# GPU won't digest images wider than 4096 on a Pi 4.
+if size[0] > 4096:
+    height = size[1] * 4096 // size[0]
```

Comment:
I wonder if we should leave this out if the test isn't displaying anything - only because it slightly obscures the main point of the example. What do you think?

Reply:
The test does display this, so it probably needs to stay in? It's in the other switch mode examples too